src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -231,6 +232,49 @@ void regclass_Core(py::module m) {
             :rtype: openvino.runtime.CompiledModel
         )");
 
+    cls.def(
+        "compile_model",
+        [](ov::Core& self,
+           const std::shared_ptr<const ov::Model>& model,
+           const RemoteContextWrapper& context,
+           const std::map<std::string, py::object>& properties) {
+            auto _properties = Common::utils::properties_to_any_map(properties);
+            py::gil_scoped_release release;
+            return self.compile_model(model, context.context, _properties);
+        },
+        py::arg("model"),
+        py::arg("context"),
+        py::arg("properties"));
+
+    cls.def(
+        "create_context",
+        [](ov::Core& self, const std::string& device_name, const std::map<std::string, py::object>& properties) {
+            auto _properties = Common::utils::properties_to_any_map(properties);
+            return RemoteContextWrapper(self.create_context(device_name, _properties));
+        },
+        py::arg("device_name"),
+        py::arg("properties"));
+
+    cls.def(
+        "create_va_context",
```

Comment:
There's no such method in c++ core API. Why do we need it in python? I'd expect that va context should be created using a wrapper which would convert some specific parameters to AnyMap, so the `create_context` method above can be used.

Reply:
This is exactly what this function is doing. Automatically doing it for the users. If someone would use directly properties it is still okay -- any time the user can provide `create_context` with custom properties.

Would you expect this?
```python
_ = core.create_context("GPU", {key: value, ...})
# and
_ = core.create_context("GPU", display)
# or
_ = core.create_context("GPU", display, -1)
```
Here is my opinion: if I were to see it first time, this is unclear what context will be produced and what is the difference. Especially that one time dict is accepted, other times some class (and integer). It is not a case like `ov.Model` creation where different options always result in the same output.

From Python perspective the function would look like this:
```python
# Keyword only arguments will apply here, as properties are missing for hardcoded VA creation:
def create_context(self, device_name: str, *, properties:dict = {}, display: VADisplayWrapper, target_tile_id: int = -1):
    # and some logic to dispatch it correctly:
    if properties is not None:
        if display is not None or target_tile_id is not None:
            # handle with warning or error
        ...
    if display is not None and ...
    # and so on...
```
or to be dispatched version which does the same thing as two functions overloads (giving it the same name). However both functions will produce different output -- depending on arguments and properties.

For this function it is more clear to have `va` in the name as it clearly states the purpose. This is additional layer that covers hardcoded values. But for clarity it makes sense to create "helper" function with different name.